File: v2/java/src/main/java/vestorly/client/model/EventContent.java
package vestorly.client.model;


import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class EventContent  {
  
  private String id = null;
  private String contentField = null;
  private String contentId = null;
  private String contentType = null;
  private String createdAt = null;
  private String slug = null;
  private String updatedAt = null;

  
  /**
   * Mongo id of event
   **/
  @ApiModelProperty(value = "Mongo id of event")
  @JsonProperty("_id")
  public String getId() {
    return id;
  }
  public void setId(String id) {
    this.id = id;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("content_field")
  public String getContentField() {
    return contentField;
  }
  public void setContentField(String contentField) {
    this.contentField = contentField;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("content_id")
  public String getContentId() {
    return contentId;
  }
  public void setContentId(String contentId) {
    this.contentId = contentId;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("content_type")
  public String getContentType() {
    return contentType;
  }
  public void setContentType(String contentType) {
    this.contentType = contentType;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("created_at")
  public String getCreatedAt() {
    return createdAt;
  }
  public void setCreatedAt(String createdAt) {
    this.createdAt = createdAt;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("slug")
  public String getSlug() {
    return slug;
  }
  public void setSlug(String slug) {
    this.slug = slug;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("updated_at")
  public String getUpdatedAt() {
    return updatedAt;
  }
  public void setUpdatedAt(String updatedAt) {
    this.updatedAt = updatedAt;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class EventContent {\n");
    
    sb.append("  id: ").append(id).append("\n");
    sb.append("  contentField: ").append(contentField).append("\n");
    sb.append("  contentId: ").append(contentId).append("\n");
    sb.append("  contentType: ").append(contentType).append("\n");
    sb.append("  createdAt: ").append(createdAt).append("\n");
    sb.append("  slug: ").append(slug).append("\n");
    sb.append("  updatedAt: ").append(updatedAt).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/vestorly/Model/EventContent.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace vestorly.model {
  [DataContract]
  public class EventContent {
    
    /* Mongo id of event */
    [DataMember(Name="_id", EmitDefaultValue=false)]
    public string id { get; set; }

    
    
    [DataMember(Name="content_field", EmitDefaultValue=false)]
    public string contentField { get; set; }

    
    
    [DataMember(Name="content_id", EmitDefaultValue=false)]
    public string contentId { get; set; }

    
    
    [DataMember(Name="content_type", EmitDefaultValue=false)]
    public string contentType { get; set; }

    
    
    [DataMember(Name="created_at", EmitDefaultValue=false)]
    public string createdAt { get; set; }

    
    
    [DataMember(Name="slug", EmitDefaultValue=false)]
    public string slug { get; set; }

    
    
    [DataMember(Name="updated_at", EmitDefaultValue=false)]
    public string updatedAt { get; set; }

    
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class EventContent {\n");
      
      sb.Append("  id: ").Append(id).Append("\n");
      
      sb.Append("  contentField: ").Append(contentField).Append("\n");
      
      sb.Append("  contentId: ").Append(contentId).Append("\n");
      
      sb.Append("  contentType: ").Append(contentType).Append("\n");
      
      sb.Append("  createdAt: ").Append(createdAt).Append("\n");
      
      sb.Append("  slug: ").Append(slug).Append("\n");
      
      sb.Append("  updatedAt: ").Append(updatedAt).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}
